domain/relation/state/relation_test.go

```diff
@@ -2600,3 +2681,27 @@ INSERT INTO unit_principal (unit_uuid, principal_uuid)
 VALUES (?,?)
 `, unitUUID1, unitUUID2)
 }
+
+func (s *relationSuite) doesRelationUnitExist(c *gc.C, relationUnitUUID corerelation.UnitUUID) bool {
+	return s.doesUUIDExist(c, "relation_unit", relationUnitUUID.String())
+}
+
+func (s *relationSuite) doesUUIDExist(c *gc.C, table, uuid string) bool {
+	found := false
+	err := s.TxnRunner().StdTxn(context.Background(), func(ctx context.Context, tx *sql.Tx) error {
+		err := tx.QueryRow(fmt.Sprintf(`
+SELECT uuid
+FROM   %s
```

Comment:
I would check this input as a precaution. Reject anything with spaces for example.

Reply:
Do we need to worry for test code that much? I would agree in production, but for a SELECT in a test what do we gain by doing that?